ztp/gitops-subscriptions/acm/groups-app.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: groups-sub
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Channel
+metadata:
+  name: groups-subscription
+  namespace: groups-sub
+  annotations:
+    apps.open-cluster-management.io/reconcile-rate: medium
+spec:
+  type: Git
+  pathname: 'https://github.com/openshift-kni/cnf-features-deploy'
```

Comment:
Is it just me, or are all these channels identical?  Do we need this duplication?  Can we share a single channel for cnf-features-deploy?

Reply:
Those channels are in different NS. They could be same or not the idea is to have every application in his own NS with its related CRs and policies.

Reply:
I'm sure there's a good reason for that, but there seems like a lot of ways that could go wrong, like making a change and forgetting one or two copies.

If we have a way to generate these identical objects based in a template that subs in the right namespace, that could go a long way to removing that maintenance burden.